src/constants/main.ts

```diff
@@ -14,13 +14,33 @@ const FORM_TABLE_FIELDS = [
         value: 'email',
     },
 ];
+const USER_TABLE_TITLE = [
```

Comment:
`USERS_TABLE_TITLES`? it's an array so please use plural

Reply:
Corrected this array name? and array COINS_TABLE_TITLE